include/range/v3/view/unique.hpp

```diff
@@ -33,27 +33,44 @@ namespace ranges
         {
             struct unique_fn
             {
-                template<typename Rng>
+            private:
+                friend view_access;
+                template<typename C,
+                        CONCEPT_REQUIRES_(!Range<C>())>
```

Comment:
We're admittedly inconsistent about whether template head continuation lines get a single additional indent (as on line 62) or are aligned with the parameters on the first line, but ideally you should pick one of those two styles and either add a space to this line or remove 3. (Also on line 48 and 54.)